tools/gendocs.sh

```diff
@@ -27,3 +27,10 @@ echo "===================================== Making HTML..."
 makeinfo --html --no-split ledger3.texi
 echo "===================================== Making PDF..."
 texi2pdf --quiet --batch ${papersize} ledger3.texi
+
+echo "===================================== Making Info..."
+makeinfo ledger-mode.texi
+echo "===================================== Making HTML..."
+makeinfo --html --no-split ledger-mode.texi
+echo "===================================== Making PDF..."
+texi2pdf --quiet --batch ${papersize} ledger-mode.texi
```

Comment:
Sure! See commit. 
But to be transparent, I did minimum effort, expecting that you will do better job in new cmakefile ! :-)